jena-fuseki2/jena-fuseki-ui/src/components/dataset/TableListing.vue

```diff
@@ -70,8 +70,8 @@
         </jena-table>
       </div>
     </div>
-    <div class="row g-0">
-      <div class="col-12">
+    <div class="row">
+      <div class="col-12 p-0">
```

Comment:
It had some padding, causing the pagination component to be slightly smaller than the table. Now they can both grow/expand to the maximum available width, reaching the same width (or the pagination may wrap to the next line if the viewport is too small, which I believe is OK)